code/modules/ftl/machinery/phase_cannon.dm

```diff
@@ -0,0 +1,95 @@
+/obj/machinery/power/shipweapon //PHYSICAL WEAPON
+	name = "phase cannon"
+	desc = "A powerful weapon designed to take down shields.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"
+	icon = 'icons/obj/96x96.dmi'
+	icon_state = "phase_cannon"
+	pixel_x = -32
+	pixel_y = -32
+	anchored = 0
+	density = 1
+
+	var/charge_rate = 200
+	var/current_charge = 0
+
+	use_power = ACTIVE_POWER_USE
+	idle_power_usage = 1000
+	active_power_usage = 20000
+
+	var/obj/item/weapon_chip/chip
```

Comment:
Needs versions with a chip in at roundstart for mapping

Reply:
I'm okay with them inserting the chip themselves